### ISSUE TITLE
lolMiner autolykos performance AMD Vega

### ISSUE THREAD

**Issue by chmichael** [Lolliedieb/lolMiner-releases#696]
Hello,
  Is it possible to improve autolykos v2 perfomance on Vega 56/64 ? 
  lolminer = 126 Mhs
  TeamRedMiner = 176 Mhs

Thank you

**Comment by jgonzis** [comment]
> Hello,
> Is it possible to improve autolykos v2 perfomance on Vega 56/64 ?
> lolminer = 126 Mhs
> TeamRedMiner = 176 Mhs
> 
> Thank you

Yes that's possible, TeamRedMiner has made and incredible job with the Vega. In the other variants lolMiner is equal or better with less Power Consumption but in VEGA  they made a great job.

**Comment by chmichael** [comment]
btw, it seems there is a trick to increase the hashrate using more VRAM (like teamredminer) on vega maybe you can apply it to nvidia cards too!

Thanks
